config.json

```diff
@@ -889,6 +889,17 @@
         "Algorithms",
         "Sorting"
       ]
+    },
+    {
+      "uuid": "ce270a34-add1-422c-bb86-53b310f05e27",
+      "slug": "transpose",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 6,
```

Comment:
My gut tells me this isn't really a 6 in terms of difficulty. This number is always so hard to judge, but because of we only have one function and a lack of tricky edge cases, I have a feeling this could be considered an easier problem. Maybe a 5? I just look at some exercises that are 7's and I see stuff like `poker` and `minesweeper`, and those are all significantly more complicated.

Reply:
I took a look at a few exercises with difficulties near the 5 level (`dominoes`, `clock`, and `difference of squares`) and 5 seems fine to me.

I agree that putting an objective measure like a number on the difficulty can be hard, especially if you're already familiar with how to solve certain sorts of problems, but not others. That can change the difficulty depending on the individual.